src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas2ApplicationStatusUpdatesReport.kt

```diff
@@ -23,7 +23,7 @@ interface Cas2ApplicationStatusUpdatesReportRepository : JpaRepository<DomainEve
          ) AS updatedAt
 
       FROM domain_events events
-      LEFT JOIN LATERAL json_array_elements(events.data -> 'eventDetails' -> 'newStatus' -> 'statusDetails') as details ON true
+      LEFT JOIN LATERAL jsonb_array_elements(events.data -> 'eventDetails' -> 'newStatus' -> 'statusDetails') as details ON true
```

Comment:
Just did a quick manual check to see that this report still works and all looks ok.

Reply:
Thanks for checking 👍